react-graphql-apollo-tutorial.md

```diff
@@ -1057,7 +1059,7 @@ const RepositoryItem = ({ ... }) => (
 );
 {{< /highlight >}}
 
-Whereas the styled Button component could be implemented like the following in the *src/Button/index.js* file. It's already extracted, because later you will make use of all its styling functionalities in other places of this application.
+The styled Button component could be implemented in the *src/Button/index.js* file. It's already extracted, because you will use its styling functionalities later in this application.
```

Comment:
"... because you will use its styling functionalities later in this application." to "... because you will use it at other places in this application later."